<log4j:event logger="com.zaxxer.hikari.pool.PoolBase" timestamp="1748692269437" level="WARN" thread="PersistedQuartzSchedulerEngineInstance_QuartzSchedulerThread">
<log4j:message><![CDATA[HikariPool-1 - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@38cd256a (No operations allowed after connection closed.). Possibly consider using a shorter maxLifetime value.]]></log4j:message>
<log4j:locationInfo class="com.zaxxer.hikari.pool.PoolBase" method="isConnectionAlive" file="PoolBase.java" line="184"/>
</log4j:event>

<log4j:event logger="com.zaxxer.hikari.pool.PoolBase" timestamp="1748699125069" level="WARN" thread="PersistedQuartzSchedulerEngineInstance_QuartzSchedulerThread">
<log4j:message><![CDATA[HikariPool-1 - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@20a31d37 (No operations allowed after connection closed.). Possibly consider using a shorter maxLifetime value.]]></log4j:message>
<log4j:locationInfo class="com.zaxxer.hikari.pool.PoolBase" method="isConnectionAlive" file="PoolBase.java" line="184"/>
</log4j:event>

<log4j:event logger="com.zaxxer.hikari.pool.HikariPool" timestamp="1748699125069" level="WARN" thread="HikariPool-1 housekeeper">
<log4j:message><![CDATA[HikariPool-1 - Thread starvation or clock leap detected (housekeeper delta=31m26s558ms6Âµs703ns).]]></log4j:message>
<log4j:locationInfo class="com.zaxxer.hikari.pool.HikariPool$HouseKeeper" method="run" file="HikariPool.java" line="788"/>
</log4j:event>

<log4j:event logger="com.zaxxer.hikari.pool.PoolBase" timestamp="1748699125070" level="WARN" thread="liferay/scheduler_dispatch-20">
<log4j:message><![CDATA[HikariPool-1 - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@6f4ef676 (No operations allowed after connection closed.). Possibly consider using a shorter maxLifetime value.]]></log4j:message>
<log4j:locationInfo class="com.zaxxer.hikari.pool.PoolBase" method="isConnectionAlive" file="PoolBase.java" line="184"/>
</log4j:event>

<log4j:event logger="org.hibernate.engine.jdbc.spi.SqlExceptionHelper" timestamp="1748699125133" level="ERROR" thread="liferay/scheduler_dispatch-20">
<log4j:message><![CDATA[HikariPool-1 - Connection is not available, request timed out after 4005493ms.]]></log4j:message>
<log4j:locationInfo class="org.hibernate.engine.jdbc.spi.SqlExceptionHelper" method="logExceptions" file="SqlExceptionHelper.java" line="142"/>
</log4j:event>

<log4j:event logger="org.hibernate.engine.jdbc.spi.SqlExceptionHelper" timestamp="1748699125133" level="ERROR" thread="liferay/scheduler_dispatch-20">
<log4j:message><![CDATA[No operations allowed after connection closed.]]></log4j:message>
<log4j:locationInfo class="org.hibernate.engine.jdbc.spi.SqlExceptionHelper" method="logExceptions" file="SqlExceptionHelper.java" line="142"/>
</log4j:event>

<log4j:event logger="com.liferay.portal.messaging.internal.ParallelDestination" timestamp="1748699125192" level="ERROR" thread="liferay/scheduler_dispatch-20">
<log4j:message><![CDATA[Unable to process message {destinationName=liferay/scheduler_dispatch, response=null, responseDestinationName=null, responseId=null, payload=null, values={GROUP_NAME=com.liferay.portal.pop.notifications.internal.scheduler.POPNotificationsSchedulerJobConfiguration, companyId=0, groupId=0, DESTINATION_NAME=liferay/scheduler_dispatch, JOB_STATE=com.liferay.portal.kernel.scheduler.JobState@59c4e602, STORAGE_TYPE=MEMORY_CLUSTERED, JOB_NAME=com.liferay.portal.pop.notifications.internal.scheduler.POPNotificationsSchedulerJobConfiguration}}]]></log4j:message>
<log4j:throwable><![CDATA[com.liferay.portal.kernel.messaging.MessageListenerException: com.liferay.portal.kernel.exception.SystemException: com.liferay.portal.kernel.dao.orm.ORMException: javax.persistence.PersistenceException: org.hibernate.exception.JDBCConnectionException: could not prepare statement
	at com.liferay.portal.scheduler.internal.SchedulerEngineHelperImpl$SchedulerJobConfigurationMessageListener.receive(SchedulerEngineHelperImpl.java:510)
	at com.liferay.portal.kernel.messaging.InvokerMessageListener.receive(InvokerMessageListener.java:62)
	at com.liferay.portal.messaging.internal.ParallelDestination$1.run(ParallelDestination.java:47)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: com.liferay.portal.kernel.exception.SystemException: com.liferay.portal.kernel.dao.orm.ORMException: javax.persistence.PersistenceException: org.hibernate.exception.JDBCConnectionException: could not prepare statement
	at com.liferay.portal.kernel.service.persistence.impl.BasePersistenceImpl.processException(BasePersistenceImpl.java:635)
	at com.liferay.portal.service.persistence.impl.CompanyPersistenceImpl.findAll(CompanyPersistenceImpl.java:1236)
	at com.liferay.portal.service.persistence.impl.CompanyPersistenceImpl.findAll(CompanyPersistenceImpl.java:1154)
	at com.liferay.portal.service.persistence.impl.CompanyPersistenceImpl.findAll(CompanyPersistenceImpl.java:1119)
	at com.liferay.portal.service.impl.CompanyLocalServiceImpl.getCompanies(CompanyLocalServiceImpl.java:717)
	at jdk.internal.reflect.GeneratedMethodAccessor865.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at com.liferay.portal.spring.aop.AopMethodInvocationImpl.proceed(AopMethodInvocationImpl.java:41)
	at com.liferay.portal.spring.transaction.TransactionInterceptor.invoke(TransactionInterceptor.java:60)
	at com.liferay.portal.spring.aop.AopMethodInvocationImpl.proceed(AopMethodInvocationImpl.java:48)
	at com.liferay.change.tracking.internal.aop.CTTransactionAdvice.invoke(CTTransactionAdvice.java:70)
	at com.liferay.portal.spring.aop.AopMethodInvocationImpl.proceed(AopMethodInvocationImpl.java:48)
	at com.liferay.portal.spring.aop.AopInvocationHandler.invoke(AopInvocationHandler.java:40)
	at com.sun.proxy.$Proxy105.getCompanies(Unknown Source)
	at com.liferay.portal.service.impl.CompanyLocalServiceImpl.forEachCompanyId(CompanyLocalServiceImpl.java:683)
	at jdk.internal.reflect.GeneratedMethodAccessor1873.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at com.liferay.portal.spring.aop.AopMethodInvocationImpl.proceed(AopMethodInvocationImpl.java:41)
	at com.liferay.portal.spring.aop.AopInvocationHandler.invoke(AopInvocationHandler.java:40)
	at com.sun.proxy.$Proxy105.forEachCompanyId(Unknown Source)
	at com.liferay.portal.pop.notifications.internal.scheduler.POPNotificationsSchedulerJobConfiguration.lambda$getJobExecutorUnsafeRunnable$1(POPNotificationsSchedulerJobConfiguration.java:59)
	at com.liferay.portal.scheduler.internal.SchedulerEngineHelperImpl$SchedulerJobConfigurationMessageListener.receive(SchedulerEngineHelperImpl.java:493)
	... 5 more
Caused by: com.liferay.portal.kernel.dao.orm.ORMException: javax.persistence.PersistenceException: org.hibernate.exception.JDBCConnectionException: could not prepare statement
	at com.liferay.portal.dao.orm.hibernate.ExceptionTranslator.translate(ExceptionTranslator.java:30)
	at com.liferay.portal.dao.orm.hibernate.QueryImpl.list(QueryImpl.java:115)
	at com.liferay.portal.dao.orm.hibernate.QueryImpl.list(QueryImpl.java:95)
	at com.liferay.portal.kernel.dao.orm.QueryUtil.list(QueryUtil.java:52)
	at com.liferay.portal.kernel.dao.orm.QueryUtil.list(QueryUtil.java:44)
	at com.liferay.portal.service.persistence.impl.CompanyPersistenceImpl.findAll(CompanyPersistenceImpl.java:1226)
	... 27 more
Caused by: javax.persistence.PersistenceException: org.hibernate.exception.JDBCConnectionException: could not prepare statement
	at org.hibernate.internal.ExceptionConverterImpl.convert(ExceptionConverterImpl.java:154)
	at org.hibernate.query.internal.AbstractProducedQuery.list(AbstractProducedQuery.java:1626)
	at com.liferay.portal.dao.orm.hibernate.QueryImpl.list(QueryImpl.java:103)
	... 31 more
Caused by: org.hibernate.exception.JDBCConnectionException: could not prepare statement
	at org.hibernate.exception.internal.SQLExceptionTypeDelegate.convert(SQLExceptionTypeDelegate.java:48)
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:37)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:113)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:186)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl.prepareQueryStatement(StatementPreparerImpl.java:151)
	at org.hibernate.loader.Loader.prepareQueryStatement(Loader.java:2122)
	at org.hibernate.loader.Loader.executeQueryStatement(Loader.java:2059)
	at org.hibernate.loader.Loader.executeQueryStatement(Loader.java:2037)
	at org.hibernate.loader.Loader.doQuery(Loader.java:956)
	at org.hibernate.loader.Loader.doQueryAndInitializeNonLazyCollections(Loader.java:357)
	at org.hibernate.loader.Loader.doList(Loader.java:2868)
	at org.hibernate.loader.Loader.doList(Loader.java:2850)
	at org.hibernate.loader.Loader.listIgnoreQueryCache(Loader.java:2682)
	at org.hibernate.loader.Loader.list(Loader.java:2677)
	at org.hibernate.hql.internal.classic.QueryTranslatorImpl.list(QueryTranslatorImpl.java:1037)
	at org.hibernate.engine.query.spi.HQLQueryPlan.performList(HQLQueryPlan.java:219)
	at org.hibernate.internal.SessionImpl.list(SessionImpl.java:1459)
	at org.hibernate.query.internal.AbstractProducedQuery.doList(AbstractProducedQuery.java:1649)
	at org.hibernate.query.internal.AbstractProducedQuery.list(AbstractProducedQuery.java:1617)
	... 32 more
Caused by: java.sql.SQLTransientConnectionException: HikariPool-1 - Connection is not available, request timed out after 4005493ms.
	at com.zaxxer.hikari.pool.HikariPool.createTimeoutException(HikariPool.java:696)
	at com.zaxxer.hikari.pool.HikariPool.getConnection(HikariPool.java:197)
	at com.zaxxer.hikari.pool.HikariPool.getConnection(HikariPool.java:162)
	at com.zaxxer.hikari.HikariDataSource.getConnection(HikariDataSource.java:128)
	at org.springframework.jdbc.datasource.LazyConnectionDataSourceProxy$LazyConnectionInvocationHandler.getTargetConnection(LazyConnectionDataSourceProxy.java:405)
	at org.springframework.jdbc.datasource.LazyConnectionDataSourceProxy$LazyConnectionInvocationHandler.invoke(LazyConnectionDataSourceProxy.java:378)
	at com.sun.proxy.$Proxy79.prepareStatement(Unknown Source)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$5.doPrepare(StatementPreparerImpl.java:149)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:176)
	... 47 more
Caused by: java.sql.SQLNonTransientConnectionException: No operations allowed after connection closed.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:110)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:63)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:73)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:73)
	at com.mysql.cj.jdbc.ConnectionImpl.setNetworkTimeout(ConnectionImpl.java:2490)
	at com.zaxxer.hikari.pool.PoolBase.setNetworkTimeout(PoolBase.java:566)
	at com.zaxxer.hikari.pool.PoolBase.isConnectionAlive(PoolBase.java:173)
	at com.zaxxer.hikari.pool.HikariPool.getConnection(HikariPool.java:186)
	... 54 more
Caused by: com.mysql.cj.exceptions.ConnectionIsClosedException: No operations allowed after connection closed.
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:61)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:105)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:151)
	at com.mysql.cj.NativeSession.checkClosed(NativeSession.java:1171)
	at com.mysql.cj.jdbc.ConnectionImpl.checkClosed(ConnectionImpl.java:576)
	at com.mysql.cj.jdbc.ConnectionImpl.setNetworkTimeout(ConnectionImpl.java:2486)
	... 57 more
]]></log4j:throwable>
<log4j:locationInfo class="com.liferay.portal.messaging.internal.ParallelDestination$1" method="run" file="ParallelDestination.java" line="50"/>
</log4j:event>

<log4j:event logger="org.quartz.core.ErrorLogger" timestamp="1748699125856" level="ERROR" thread="PersistedQuartzSchedulerEngineInstance_QuartzSchedulerThread">
<log4j:message><![CDATA[An error occurred while scanning for the next triggers to fire.]]></log4j:message>
<log4j:throwable><![CDATA[org.quartz.JobPersistenceException: Couldn't acquire next trigger: HikariPool-1 - Connection is not available, request timed out after 4493727ms. [See nested exception: java.sql.SQLTransientConnectionException: HikariPool-1 - Connection is not available, request timed out after 4493727ms.]
	at org.quartz.impl.jdbcjobstore.JobStoreSupport.acquireNextTrigger(JobStoreSupport.java:2827)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport$41.execute(JobStoreSupport.java:2755)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport.executeInNonManagedTXLock(JobStoreSupport.java:3798)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport.acquireNextTriggers(JobStoreSupport.java:2751)
	at org.quartz.core.QuartzSchedulerThread.run(QuartzSchedulerThread.java:264)
Caused by: java.sql.SQLTransientConnectionException: HikariPool-1 - Connection is not available, request timed out after 4493727ms.
	at com.zaxxer.hikari.pool.HikariPool.createTimeoutException(HikariPool.java:696)
	at com.zaxxer.hikari.pool.HikariPool.getConnection(HikariPool.java:197)
	at com.zaxxer.hikari.pool.HikariPool.getConnection(HikariPool.java:162)
	at com.zaxxer.hikari.HikariDataSource.getConnection(HikariDataSource.java:128)
	at org.springframework.jdbc.datasource.LazyConnectionDataSourceProxy$LazyConnectionInvocationHandler.getTargetConnection(LazyConnectionDataSourceProxy.java:405)
	at org.springframework.jdbc.datasource.LazyConnectionDataSourceProxy$LazyConnectionInvocationHandler.invoke(LazyConnectionDataSourceProxy.java:378)
	at com.sun.proxy.$Proxy79.prepareStatement(Unknown Source)
	at jdk.internal.reflect.GeneratedMethodAccessor110.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.quartz.impl.jdbcjobstore.AttributeRestoringConnectionInvocationHandler.invoke(AttributeRestoringConnectionInvocationHandler.java:73)
	at com.sun.proxy.$Proxy1133.prepareStatement(Unknown Source)
	at org.quartz.impl.jdbcjobstore.StdJDBCDelegate.selectTriggerToAcquire(StdJDBCDelegate.java:2557)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport.acquireNextTrigger(JobStoreSupport.java:2778)
	... 4 more
Caused by: java.sql.SQLNonTransientConnectionException: No operations allowed after connection closed.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:110)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:63)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:73)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:73)
	at com.mysql.cj.jdbc.ConnectionImpl.setNetworkTimeout(ConnectionImpl.java:2490)
	at com.zaxxer.hikari.pool.PoolBase.setNetworkTimeout(PoolBase.java:566)
	at com.zaxxer.hikari.pool.PoolBase.isConnectionAlive(PoolBase.java:173)
	at com.zaxxer.hikari.pool.HikariPool.getConnection(HikariPool.java:186)
	at com.zaxxer.hikari.pool.HikariPool.getConnection(HikariPool.java:162)
	at com.zaxxer.hikari.HikariDataSource.getConnection(HikariDataSource.java:128)
	at org.springframework.jdbc.datasource.LazyConnectionDataSourceProxy$LazyConnectionInvocationHandler.getTargetConnection(LazyConnectionDataSourceProxy.java:405)
	at org.springframework.jdbc.datasource.LazyConnectionDataSourceProxy$LazyConnectionInvocationHandler.invoke(LazyConnectionDataSourceProxy.java:378)
	at com.sun.proxy.$Proxy79.prepareStatement(Unknown Source)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$5.doPrepare(StatementPreparerImpl.java:149)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl$StatementPreparationTemplate.prepareStatement(StatementPreparerImpl.java:176)
	at org.hibernate.engine.jdbc.internal.StatementPreparerImpl.prepareQueryStatement(StatementPreparerImpl.java:151)
	at org.hibernate.loader.Loader.prepareQueryStatement(Loader.java:2122)
	at org.hibernate.loader.Loader.executeQueryStatement(Loader.java:2059)
	at org.hibernate.loader.Loader.executeQueryStatement(Loader.java:2037)
	at org.hibernate.loader.Loader.doQuery(Loader.java:956)
	at org.hibernate.loader.Loader.doQueryAndInitializeNonLazyCollections(Loader.java:357)
	at org.hibernate.loader.Loader.doList(Loader.java:2868)
	at org.hibernate.loader.Loader.doList(Loader.java:2850)
	at org.hibernate.loader.Loader.listIgnoreQueryCache(Loader.java:2682)
	at org.hibernate.loader.Loader.list(Loader.java:2677)
	at org.hibernate.hql.internal.classic.QueryTranslatorImpl.list(QueryTranslatorImpl.java:1037)
	at org.hibernate.engine.query.spi.HQLQueryPlan.performList(HQLQueryPlan.java:219)
	at org.hibernate.internal.SessionImpl.list(SessionImpl.java:1459)
	at org.hibernate.query.internal.AbstractProducedQuery.doList(AbstractProducedQuery.java:1649)
	at org.hibernate.query.internal.AbstractProducedQuery.list(AbstractProducedQuery.java:1617)
	at com.liferay.portal.dao.orm.hibernate.QueryImpl.list(QueryImpl.java:103)
	at com.liferay.portal.dao.orm.hibernate.QueryImpl.list(QueryImpl.java:95)
	at com.liferay.portal.kernel.dao.orm.QueryUtil.list(QueryUtil.java:52)
	at com.liferay.portal.kernel.dao.orm.QueryUtil.list(QueryUtil.java:44)
	at com.liferay.portal.service.persistence.impl.CompanyPersistenceImpl.findAll(CompanyPersistenceImpl.java:1226)
	at com.liferay.portal.service.persistence.impl.CompanyPersistenceImpl.findAll(CompanyPersistenceImpl.java:1154)
	at com.liferay.portal.service.persistence.impl.CompanyPersistenceImpl.findAll(CompanyPersistenceImpl.java:1119)
	at com.liferay.portal.service.impl.CompanyLocalServiceImpl.getCompanies(CompanyLocalServiceImpl.java:717)
	at jdk.internal.reflect.GeneratedMethodAccessor865.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at com.liferay.portal.spring.aop.AopMethodInvocationImpl.proceed(AopMethodInvocationImpl.java:41)
	at com.liferay.portal.spring.transaction.TransactionInterceptor.invoke(TransactionInterceptor.java:60)
	at com.liferay.portal.spring.aop.AopMethodInvocationImpl.proceed(AopMethodInvocationImpl.java:48)
	at com.liferay.change.tracking.internal.aop.CTTransactionAdvice.invoke(CTTransactionAdvice.java:70)
	at com.liferay.portal.spring.aop.AopMethodInvocationImpl.proceed(AopMethodInvocationImpl.java:48)
	at com.liferay.portal.spring.aop.AopInvocationHandler.invoke(AopInvocationHandler.java:40)
	at com.sun.proxy.$Proxy105.getCompanies(Unknown Source)
	at com.liferay.portal.service.impl.CompanyLocalServiceImpl.forEachCompanyId(CompanyLocalServiceImpl.java:683)
	at jdk.internal.reflect.GeneratedMethodAccessor1873.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at com.liferay.portal.spring.aop.AopMethodInvocationImpl.proceed(AopMethodInvocationImpl.java:41)
	at com.liferay.portal.spring.aop.AopInvocationHandler.invoke(AopInvocationHandler.java:40)
	at com.sun.proxy.$Proxy105.forEachCompanyId(Unknown Source)
	at com.liferay.portal.pop.notifications.internal.scheduler.POPNotificationsSchedulerJobConfiguration.lambda$getJobExecutorUnsafeRunnable$1(POPNotificationsSchedulerJobConfiguration.java:59)
	at com.liferay.portal.scheduler.internal.SchedulerEngineHelperImpl$SchedulerJobConfigurationMessageListener.receive(SchedulerEngineHelperImpl.java:493)
	at com.liferay.portal.kernel.messaging.InvokerMessageListener.receive(InvokerMessageListener.java:62)
	at com.liferay.portal.messaging.internal.ParallelDestination$1.run(ParallelDestination.java:47)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: com.mysql.cj.exceptions.ConnectionIsClosedException: No operations allowed after connection closed.
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:61)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:105)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:151)
	at com.mysql.cj.NativeSession.checkClosed(NativeSession.java:1171)
	at com.mysql.cj.jdbc.ConnectionImpl.checkClosed(ConnectionImpl.java:576)
	at com.mysql.cj.jdbc.ConnectionImpl.setNetworkTimeout(ConnectionImpl.java:2486)
	... 57 more
]]></log4j:throwable>
<log4j:locationInfo class="org.quartz.core.ErrorLogger" method="schedulerError" file="QuartzScheduler.java" line="2361"/>
</log4j:event>

<log4j:event logger="com.zaxxer.hikari.pool.PoolBase" timestamp="1748699153279" level="WARN" thread="liferay/scheduler_dispatch-32">
<log4j:message><![CDATA[HikariPool-1 - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@5f303206 (No operations allowed after connection closed.). Possibly consider using a shorter maxLifetime value.]]></log4j:message>
<log4j:locationInfo class="com.zaxxer.hikari.pool.PoolBase" method="isConnectionAlive" file="PoolBase.java" line="184"/>
</log4j:event>

<log4j:event logger="com.zaxxer.hikari.pool.PoolBase" timestamp="1748699153280" level="WARN" thread="liferay/scheduler_dispatch-20">
<log4j:message><![CDATA[HikariPool-1 - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@65ac8792 (No operations allowed after connection closed.). Possibly consider using a shorter maxLifetime value.]]></log4j:message>
<log4j:locationInfo class="com.zaxxer.hikari.pool.PoolBase" method="isConnectionAlive" file="PoolBase.java" line="184"/>
</log4j:event>

<log4j:event logger="com.zaxxer.hikari.pool.PoolBase" timestamp="1748699153290" level="WARN" thread="liferay/scheduler_dispatch-30">
<log4j:message><![CDATA[HikariPool-1 - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@1f5aadc7 (No operations allowed after connection closed.). Possibly consider using a shorter maxLifetime value.]]></log4j:message>
<log4j:locationInfo class="com.zaxxer.hikari.pool.PoolBase" method="isConnectionAlive" file="PoolBase.java" line="184"/>
</log4j:event>

<log4j:event logger="com.zaxxer.hikari.pool.HikariPool" timestamp="1748699155082" level="WARN" thread="HikariPool-1 housekeeper">
<log4j:message><![CDATA[HikariPool-1 - Thread starvation or clock leap detected (housekeeper delta=1h16m49s35ms916Âµs706ns).]]></log4j:message>
<log4j:locationInfo class="com.zaxxer.hikari.pool.HikariPool$HouseKeeper" method="run" file="HikariPool.java" line="788"/>
</log4j:event>

<log4j:event logger="com.zaxxer.hikari.pool.HikariPool" timestamp="1748714079168" level="WARN" thread="HikariPool-1 housekeeper">
<log4j:message><![CDATA[HikariPool-1 - Thread starvation or clock leap detected (housekeeper delta=7m33s205ms550Âµs460ns).]]></log4j:message>
<log4j:locationInfo class="com.zaxxer.hikari.pool.HikariPool$HouseKeeper" method="run" file="HikariPool.java" line="788"/>
</log4j:event>

<log4j:event logger="com.zaxxer.hikari.pool.HikariPool" timestamp="1748721239519" level="WARN" thread="HikariPool-1 housekeeper">
<log4j:message><![CDATA[HikariPool-1 - Thread starvation or clock leap detected (housekeeper delta=1h27m18s361ms550Âµs622ns).]]></log4j:message>
<log4j:locationInfo class="com.zaxxer.hikari.pool.HikariPool$HouseKeeper" method="run" file="HikariPool.java" line="788"/>
</log4j:event>

